scripts/test-purge-all.sh

```diff
@@ -0,0 +1,1330 @@
+#!/bin/bash
```

Comment:
Having the script is great but am not sure to what extent I'd like to consider it fully reliable yet. Can we add it to an /experimental folder under scripts?